client/app/iplb/configuration/iplb-configuration.html

```diff
@@ -1,27 +1,40 @@
 <section class="cui-page__content">
     <cui-page-content-title text="'iplb_configuration_title' | translate"></cui-page-content-title>
     <p data-translate="iplb_configuration_intro"></p>
-
-    <button type="button"
-        class="oui-button oui-button_secondary"
-        data-translate="iplb_configuration_action_apply"
-        data-ng-click="ctrl.applyChanges()"
-        data-ng-disabled="!ctrl.selectedZones.length"></button>
-
-    <oui-spinner data-ng-show="ctrl.zones.loading"></oui-spinner>
-    <oui-table data-ng-hide="ctrl.zones.loading"
-        rows="ctrl.zones.data"
-        on-selection-change="ctrl.onSelectionChange($selection)">
-        <column title="'iplb_configuration_datacenter' | translate" property="name" sortable="asc"></column>
-        <column title="'iplb_configuration_progress' | translate"
-            sortable
-            class="oui-table__cell_component">
+    <oui-datagrid data-rows="ctrl.zones.data">
+        <extra-top>
+            <button type="button"
+                class="oui-button oui-button_secondary"
+                data-translate="iplb_configuration_action_apply"
+                data-ng-click="ctrl.applyChanges()"
+                data-ng-disabled="!ctrl.selectedZones.length"></button>
```

Comment:
`!ctrl.selectedZones || !ctrl.selectedZones.length` ?

Reply:
I don't think we need to add !ctrl.selectedZones. Because selectedZones will be an array always with or without items in it. It can be an empty array but it is an array always.

Reply:
:+1: alright